ui/src/utils/gqlToApi.js

```diff
@@ -0,0 +1,23 @@
+import { omit } from 'ramda';
+import cloneDeep from 'lodash.clonedeep';
+import removeKeys from './removeKeys';
+import { TASK_ADDED_FIELDS } from './constants';
+
+/**
+ * Given a task in the format embedded in graphql queries, convert it
+ * into a "normal" task payload.
+ */
```

Comment:
Is the graphql task format documented somewhere? If so it might be nice to cross reference it here.

Reply:
The graphql schema itself "documents" it -- but it's not a public API, so we don't publish that in any way.